scarlet2/spectrum.py

```diff
@@ -1,12 +1,21 @@
 import equinox as eqx
 from .bbox import Box
 from .module import Parameter, Module
+import jax.numpy as jnp
 
 class Spectrum(Module):
     bbox: Box = eqx.static_field()
 
+class Specind(Module):
+    bbox: Box = eqx.static_field()
 
 class ArraySpectrum(Spectrum, Parameter):
     def __init__(self, *args, **kwargs):
         Parameter.__init__(self, *args, **kwargs)
-        self.bbox = Box(self.value.shape)
\ No newline at end of file
+        self.bbox = Box(self.value.shape)
+
+class StaticArraySpectrum(Spectrum, Parameter):
+    def __init__(self, *args, **kwargs):
+        Parameter.__init__(self, *args, **kwargs)  
+        self.bbox = Box(jnp.repeat(self.value,self.repeats).shape)
```

Comment:
I'm confused: 
* I don't see a definition of `self.repeats`.
* How does manipulating the `bbox` get you a source that has the same spectrum across multiple observations/channels. I think this is related to the `specind` situation above, where you seem to multiply the spectra with a slicing operation.